portfolio/src/main/webapp/script.js

```diff
@@ -45,3 +46,17 @@ function createAnyElement(tag, text) {
   textElement.innerText = text;
   return textElement;
 }
+
+/**
+ * Deletes all comments from the database.
+ */
+async function deleteComments() {
+  var confirmation = confirm("Are you sure you want to delete all comments?");
```

Comment:
Let's replace `var` with `const`. We typically try to use only `const` or `let`:

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
Good to know - done.